cfme/scripting/appliance.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""Script to encrypt config files.
+
+Usage:
+
+   scripts/encrypt_conf.py confname1 confname2 ... confnameN
+   scripts/encrypt_conf.py credentials
+"""
+
+from cached_property import cached_property
+import click
+
+from functools import partial
+
+from utils.appliance import IPAppliance
+
+
+@click.group(help='Helper commands for appliances')
+def main():
+    """Main appliance group"""
+    pass
+
+
+@main.command('reboot', help='Reboots a provider')
+@click.argument('appliance_ip')
+@click.option('--wait_for_ui', is_flag=True, default=True)
+def reboot_appliance(appliance_ip, wait_for_ui):
```

Comment:
yes, i jsut wanted to comment on it as its a really nice followup after merge

it can also be used in other places where we need the appliance for the cli